proposed/cache.md

```diff
@@ -293,11 +293,13 @@ interface CacheItemPoolInterface
      *
      * @param string $key
      *   The key for which to return the corresponding Cache Item.
-     * @return \Psr\Cache\CacheItemInterface
-     *   The corresponding Cache Item.
-     * @throws \Psr\Cache\InvalidArgumentException
+     *
+     * @throws InvalidArgumentException
      *   If the $key string is not a legal value a \Psr\Cache\InvalidArgumentException
      *   MUST be thrown.
+     *
+     * @return CacheItemInterface
+     *   The corresponding Cache Item.
```

Comment:
Why rearrange these?  I'm not aware of any general standard on the order here, and more often than not what I see in practice is @throws at the very end.

Does PSR-5 have a statement on the matter?